bitwarden_license/src/CommCore/CommCore.csproj

```diff
@@ -1,9 +1,5 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
```

Comment:
I think we should still specify target framework and just bump to 6.0

Reply:
The target framework is specified in the `Directory.Build.props` file so that all projects will get that target framework. We don't specify it in most of the project in the repo and the ones I removed it from are the only exceptions.

Reply:
ah, my mistake. Ignore all these, then